plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftConnectorTest.java

```diff
@@ -116,6 +118,22 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         };
     }
 
+    @Test
+    void testJdbcFlow()
```

Comment:
How is this different from `testUnloadFlowFallbackToJdbc` ?
Maybe this should test a query that is supported by UNLOAD but run with unload session property disabled

Reply:
removed this test